lib/dry/behaviour/black_tie.rb

```diff
@@ -1,4 +1,8 @@
 module Dry
+  # rubocop:disable Style/VariableName
+  # rubocop:disable Style/AsciiIdentifiers
+  # rubocop:disable Style/MultilineBlockChain
+  # rubocop:disable Style/EmptyCaseCondiiton
```

Comment:
`EmptyCaseCondiiton` => `EmptyCaseCondition` (otherwise it's not really disabling anything...)